static/css/responsive.css

```
@@ -196,7 +196,7 @@
 /* ── High contrast mode ─── */
 @media (prefers-contrast: high) {
     * {
-        border-color: currentcolor !important;
+        border-color: currentColor !important;
```

Stylelint (17.12.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)